src/kernel/boot.c

```diff
@@ -655,13 +655,7 @@ BOOT_CODE bool_t create_untypeds(cap_t root_cnode_cap,
         region_t reg = paddr_to_pptr_reg((p_region_t) {
             start, CONFIG_PADDR_USER_DEVICE_TOP
         });
-        /*
-         * The auto-generated bitfield code will get upset if the
-         * end pptr is larger than the maximum pointer size for this architecture.
-         */
-        if (reg.end > PPTR_TOP) {
-            reg.end = PPTR_TOP;
-        }
```

Comment:
Good to see this is gone here. I have been wondering for some time now what this was for.

Reply:
Aha, finally found something that this was protecting: `TX2_debug_hyp_gcc_64 FAILED`.  I'll take a look at it tomorrow.

Reply:
Now fixed with https://github.com/seL4/seL4/pull/590